spring/pim-tool-back/src/main/java/vn/elca/training/validator/ValidStatusValidator.java

```diff
@@ -0,0 +1,13 @@
+package vn.elca.training.validator;
+
+import vn.elca.training.enumaration.Status;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class ValidStatusValidator implements ConstraintValidator<ValidStatus, Status> {
+    @Override
+    public boolean isValid(Status value, ConstraintValidatorContext context) {
+        return value != null && (value == Status.NEW || value == Status.PLA || value == Status.INP || value == Status.FIN);
```

Comment:
[N] Just use Status.valueOf and catch its exception in case invalid status. It will make it easier to add new Enum in the future.